### ISSUE TITLE
System update: k is not defined

### ISSUE THREAD

**Issue by orkaboy** [drewg13/foundryvtt-band-of-blades#36]
When I run system update (on Band of Blades v1.8.3), I get this (note that the same error is present for the Scum and Villainy system):

![image](https://user-images.githubusercontent.com/1805679/180315321-f6b94016-7e8f-4b5f-9dd3-579fbac1f5ea.png)

I'm assuming this is an update to Foundry V10 issue (I'm currently running Stable 9.269). What is the recommended Foundry update channel/version that you are currently running? Also note that the PbtA and DW appears to have similar issues, but they fail more gracefully.

I got a popup when updating to v1.8.3 that the path to the manifest had changed, not sure if related but thought I'd mention it.

**Comment by orkaboy** [comment]
Just to not send you on a while goose chase, I saw this for some modules as well (not just your systems). I read this from theripper93's Discord for the exact same error message:

> Note: Updating my modules will give you this error while you are on V9 if a V10 version is present, you can safely ignore it

**Comment by drewg13** [comment]
I was aware of that, although I thought I included the V9 fields needed to not trigger that error.  As you guessed, those are V10 updates, but if you look at the changelogs, there's not that much I've added that would be worth upgrading for so far.  Maintaining backwards compatibility was unpleasant and needlessly complex with all of the Document API changes, so releasing V10-only versions from here out made the most sense to me.  The code is pretty stable and there's nothing major I was planning to add.
